kentikapi/internal/api_payloads/device.go

```diff
@@ -402,7 +434,7 @@ func payloadToDevicePlan(p devicePlanPayload) (models.DevicePlan, error) {
 
 // labelIDPayload represents JSON ApplyLabels.LabelID payload as it is transmitted to KentikAPI.
 type labelIDPayload struct {
-	ID int `json:"id"`
```

Comment:
I think the ID type here should stay as integer, as this is what Kentik API expects in Apply Labels request
https://api.kentik.com/api/tester/v5#device-PUT-Device-apply-labels